generate_R_table.py

```diff
@@ -1,3 +1,4 @@
+#!/Users/antonmalko/anaconda/bin/ python
```

Comment:
Oh, was that bug report about the script not being able to process the DA1 files produced by Adrian Staub's "robodoc"? 
Even if no, the ability to recognize EyeDoctor and robodoc files would be a useful thing to add at some point (the DA1 files produced by the latter miss a couple of columns, which messes your script up, obviously). Maybe I'll dig through this in summer